test/bdd/pkg/internal/httputil/httputil.go

```diff
@@ -161,13 +161,6 @@ func WithBody(val []byte) Opt {
 	}
 }
 
-// WithBearerToken specifies an authorization bearer token.
-func WithBearerToken(token string) Opt {
-	return func(o *options) {
-		o.bearerToken = token
-	}
-}
-
```

Comment:
if this is not needed, remove plz `bearerToken` from `options` struct and behavior related to setting `Authorization` header in `DoRequest` method

Reply:
removed